operations/image_management/Configure_IMS_to_validate_rpms.md

```diff
@@ -68,14 +92,113 @@ Configuring the Image Management Service (IMS) to validate the GPG signatures of
     [...]
     ```
 
-1. Create a `Dockerfile` to create a new `cray-ims-kiwi-ng-opensuse-x86_64-builder` image.
+1. (`ncn-mw#`) (CSM 1.5.0 and later) Obtain a copy of the `armentry.sh` script from `cray-ims-kiwi-ng-opensuse-x86_64-builder`.
+
+   ```bash
+   podman run -it --entrypoint "" --rm registry.local/artifactory.algol60.net/csm-docker/stable/cray-ims-kiwi-ng-opensuse-x86_64-builder:${KIWI_VERSION} cat /scripts/armentry.sh | tee armentry.sh
+   ```
+
+1. (`ncn-mw#`) (CSM 1.5.0 and later) Set the correct permissions on the script file.
+
+    ```bash
+   chmod 755 armentry.sh
+   ```
+
+    Verify the correct permissions:
+
+    ```bash
+    ls -la armentry.sh
+    ```
+
+    Expected output:
+
+    ```text
+    -rwxr-xr-x 1 root root 8955 Jul 24 15:27 armentry.sh
+    ```
+
+1. (`ncn-mw#`) (CSM 1.5.0 and later) Modify the `armentry.sh` script to pass the signing key to the `kiwi-ng` command.
+
+    ```bash
+    cat armentry.sh
+    ```
+
+    Example output:
+
+    ```text
+    [...]
+
+    # Call kiwi to build the image recipe. Note that the command line --add-bootstrap-package
+    # causes kiwi to install the cray-ca-cert RPM into the image root.
+    kiwi-ng $DEBUG_FLAGS --logfile=$PARAMETER_FILE_KIWI_LOGFILE --type tbz system build --description $RECIPE_ROOT_PARENT \
+    --target $IMAGE_ROOT_PARENT --add-bootstrap-package file:///mnt/ca-rpm/cray_ca_cert-1.0.1-1.x86_64.rpm \
+    --signing-key /signing-keys/my-signing-key.asc   # <--- ADD SIGNING-KEY FILE
+
+    [...]
+    ```
+
+1. (`ncn-mw#`) (CSM 1.5.2 and later) Obtain a copy of the `remote_build_entrypoint.sh` script from `cray-ims-kiwi-ng-opensuse-x86_64-builder`.
+
+   ```bash
+   podman run -it --entrypoint "" --rm registry.local/artifactory.algol60.net/csm-docker/stable/cray-ims-kiwi-ng-opensuse-x86_64-builder:${KIWI_VERSION} cat /scripts/remote_build_entrypoint.sh | tee remote_build_entrypoint.sh
+   ```
+
+1. (`ncn-mw#`) (CSM 1.5.2 and later) Set the correct permissions on the script file.
+
+    ```bash
+   chmod 755 remote_build_entrypoint.sh
+   ```
+
+    Verify the correct permissions:
+
+    ```bash
+    ls -la remote_build_entrypoint.sh
+    ```
+
+    Expected output:
+
+    ```text
+    -rwxr-xr-x 1 root root 8955 Jul 24 15:27 remote_build_entrypoint.sh
+    ```
+
+1. (`ncn-mw#`) (CSM 1.5.2 and later) Modify the `remote_build_entrypoint.sh` script to pass the signing key to the `kiwi-ng` command.
```

Comment:
Since this is going into the CSM 1.6 docs, isn't the "CSM 1.5.2 and later" part unnecessary?
(Obviously it would be needed for the CSM 1.5 backport, of course)